package/repos/codeberg.py

```diff
@@ -140,11 +140,14 @@ def fetch_commits(self, package):
                 if not created:
                     break
 
-                collaborators.append(commit.user)
+                if commit.user and commit.user != "":
+                    collaborators.append(commit.user)
             except Commit.MultipleObjectsReturned:
                 continue
 
-        package.participants = ",".join(set(collaborators))
+        if len(collaborators) > 0:
+            package.participants = ",".join(set(collaborators))
```

Comment:
A `package` is only updated when there are `collaborators` present, so, I think we should only call `save()` if that condition is `True`

```suggestion
        if collaborators:
            package.participants = ",".join(set(collaborators))
            package.save()
```

Reply:
I assumed syncing commits falls under the notion of a "repository fetch" and `Package.last_fetched` should be updated, even if participants don't change?

Reply:
`last_fetched` should be updated per run. We use it to order the Packages when we are fetching for updates which solved an issue where some packages in the list where never getting updated because of another bug.